config.py

```diff
@@ -34,3 +34,6 @@
   'Alice': '221b Baker Street, London NW1 6XE',
   'Bob': 'Buckingham Palace, London SW1A 1AA'
 }
+
+house_member_email_addresses = {
```

Comment:
It would be great if there was an example in place for people to use and adapt, as with Alice and Bob's address.